src/nvim/api/extmark.c

```diff
@@ -404,6 +404,10 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id, Object start, Object e
 ///                   for left). Defaults to false.
 ///               - priority: a priority value for the highlight group. For
 ///                   example treesitter highlighting uses a value of 100.
+///               - strict: boolean that indicates extmark should be placed
+///                   even if the line or column value is past the end of the
+///                   buffer or end of the line respectively. Defaults to true.
```

Comment:
I'm confused by this wording, does `strict = true` mean that the extmark will be placed even if the col value is past the end of line? Or is that the case when `strict = false`? I would expect the latter but this implies the former.

Reply:
It's just bad wording, I'll fix it.